pydantic_ai_slim/pydantic_ai/toolsets/_callable.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from dataclasses import replace
+from typing import TYPE_CHECKING, Any
+
+from pydantic_core import SchemaValidator
+
+from .._run_context import AgentDepsT, RunContext
+from ..messages import ToolCallPart
+from . import AbstractToolset
+
+if TYPE_CHECKING:
+    pass
```

Comment:
suggestion: remove as its not needed, annoyingly ruff leaves them when it removes the last entry.